### PR TITLE
Implement Azure Blob Storage Adapter

### PR DESCRIPTION
**Pull request by Brammm**
PR remade from https://github.com/thephpleague/flysystem-azure-blob-storage/pull/23

### CODE REVIEW

File: src/AzureBlobStorage/.travis.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+language: php
```
Comment (Nyholm) [inline src/AzureBlobStorage/.travis.yml:1]: This file is not needed.

File: src/AzureBlobStorage/composer.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        }
+    },
+    "require": {
+        "php": "^7.2 || ^8.0",
+        "league/flysystem": "^2.0.0",
+        "microsoft/azure-storage-blob": "^1.1",
+        "guzzlehttp/psr7": "^1.5"
+    },
+    "require-dev": {
+        "phpunit/phpunit": "^5.7"
```
Comment (Nyholm) [inline src/AzureBlobStorage/composer.json:18]: Is this really needed? If so, Lets do PHPUnit 9

File: src/AzureBlobStorage/composer.json [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    "autoload": {
+        "psr-4": {
+            "League\\Flysystem\\AzureBlobStorage\\": ""
+        }
+    },
+    "require": {
+        "php": "^7.2 || ^8.0",
+        "league/flysystem": "^2.0.0",
+        "microsoft/azure-storage-blob": "^1.1",
+        "guzzlehttp/psr7": "^1.5"
```
Comment (Nyholm) [inline src/AzureBlobStorage/composer.json:15]: We dont need this package. We are just using one method. 

And also, nyholm/psr7 is better =)

**Review by Nyholm — COMMENTED** [review]
Thank you for this PR.

**Comment by Warxcell** [comment]
Up?

**Comment by Nyholm** [comment]
@Warxcell, that is a little strange message. What I assume that you mean is something like this:

> Thank you @Brammm for making this PR. I also like to see this feature. Do you plan to finish the PR or do you need some help?

Right?

**Comment by Warxcell** [comment]
> @Warxcell, that is a little strange message. What I assume that you mean is something like this:
> 
> > Thank you @Brammm for making this PR. I also like to see this feature. Do you plan to finish the PR or do you need some help?
> 
> Right?

Yeah. Or get my PR: https://github.com/thephpleague/flysystem-azure-blob-storage/pull/25

**Comment by scadh** [comment]
Is this still under development and/or consideration for merge? I'd like to see an Azure implementation for v2 as well, but my need is immediate. Is it held up due to v8 unit test failure?

**Pull request closed without merging**